utils/database.js

```diff
@@ -0,0 +1,37 @@
+const bcrypt = require('bcrypt');
+
+const users = [

```

Comment:
the data shouldn't be store in js files.

Reply:
and also shouldn't be in utils

Reply:
I keep it in js file coz it's king of mock data, in next HW i'll remove it and set database